AEPCore/Sources/eventhub/EventHub.swift

```diff
@@ -214,7 +214,7 @@ final class EventHub {
     /// Retrieves the `SharedState` for a specific extension
     /// - Parameters:
     ///   - extensionName: An extension name whose `SharedState` will be returned
-    ///   - event: If not nil, will retrieve the `SharedState` that corresponds with this event's version. If nil will return the earliest `SharedState`
+    ///   - event: If not nil, will retrieve the `SharedState` that corresponds with this event's version or latest if not yet versioned. If event is nil will return the earliest `SharedState`
```

Comment:
In Android and Objc core we return latest shared state when event is nil. I think we should make them consistent in that situation - this may be considered as a bug fix as this functionality existed before.

Reply:
It's intended to return the earliest, it's the best way around the fact we don't have a boot even to fetch the latest. 
https://github.com/adobe/aepsdk-core-ios/issues/563

Reply:
But this breaks this contract on the public getSharedState API https://github.com/adobe/aepsdk-core-ios/blob/main/AEPCore/Sources/eventhub/ExtensionRuntime.swift#L65

Reply:
@emdobrin good catch, we should update the comment in the runtime class

Reply:
I am concerned that this will cause issues for developers implementing both iOS and Android extensions, as the behavior is not consistent. The intention with null event for getSharedState was originally this, to always get latest, not earliest.

Reply:
Thanks for bringing this up @emdobrin. I had this same question when I was reviewing the Target swift code where we fetch and cache the latest available config.